ui/src/testing/stubs/concept-sets-api-stub.ts

```diff
@@ -0,0 +1,133 @@
+import {ConceptsServiceStub} from './concepts-service-stub';
+
+import {
+  ConceptSet,
+  ConceptSetListResponse,
+  Domain,
+  UpdateConceptSetRequest
+} from 'generated/fetch';
+import {ConceptSetsApi, EmptyResponse} from 'generated/fetch/api';
+
+export class ConceptSetsApiStub extends ConceptSetsApi {
+  public conceptSets?: ConceptSet[];
+  // TODO when this piece is converted
+  // public conceptsStub?: ConceptsServiceStub;
+
+  constructor() {
+    super(undefined, undefined, (..._: any[]) => { throw Error('cannot fetch in tests'); });
+
+    this.conceptSets = ConceptSetsApiStub.stubConceptSets();
+    // this.conceptsStub = new ConceptsServiceStub();
+
+  }
+
+  static stubConceptSets(): ConceptSet[] {
+    return [
+      {
+        id: 345,
+        name: 'Mock Concept Set',
+        description: 'Mocked for tests',
+        domain: Domain.CONDITION,
+        lastModifiedTime: new Date().getTime() - 8000
+      },  {
+        id: 346,
+        name: 'Mock Concept Set Measurement',
+        description: 'Mocked for tests',
+        domain: Domain.MEASUREMENT,
+        lastModifiedTime: new Date().getTime()
+      },  {
+        id: 347,
+        name: 'Mock Concept Set for condition',
+        description: 'Mocked for tests',
+        domain: Domain.CONDITION,
+        lastModifiedTime: new Date().getTime() - 2000
+      }
+    ];
+  }
+
+  private mustFindConceptSet(conceptSetId: number): ConceptSet {
+    const target = this.conceptSets.find(cs => cs.id === conceptSetId);
+    if (!target) {
+      throw Error(`concept set ${conceptSetId} not found`);
+    }
+    return target;
+  }
+
+  public getConceptSetsInWorkspace(
+    workspaceNamespace: string, workspaceId: string): Promise<ConceptSetListResponse> {
+    return new Promise<ConceptSetListResponse>(resolve => {
+      setTimeout(() => {
+        resolve({items: this.conceptSets});
+      }, 0);
+    });
+  }
+
+  public updateConceptSet(
+    workspaceNamespace: string, workspaceId: string, conceptSetId: number,
+    req: ConceptSet): Promise<ConceptSet> {
+    return new Promise<ConceptSet>(resolve => {
+      setTimeout(() => {
+        const target = this.mustFindConceptSet(conceptSetId);
+        target.name = req.name;
+        target.description = req.description;
+        resolve(target);
+      }, 0);
+    });
+  }
+
+  // public createConceptSet(
```

Comment:
nit: I prefer to avoid commented out code if possible. It should be easy enough to look at the service stub or git history once we need to port this.